src/github/handlers/push-event.ts

```diff
@@ -28,7 +29,7 @@ function constructErrorBody(
         }
         const node = doc.getIn(path, true) as Node;
         const linePosStart = lineCounter.linePos(node?.range?.[0] || 0);
-        body.push(`> https://github.com/${repository.owner?.login}/${repository.name}/blob/${after}/${CONFIG_FULL_PATH}#L${linePosStart.line}`);
+        body.push(`> https://github.com/${repository.owner?.login}/${repository.name}/blob/${after}/${configPath}#L${linePosStart.line}`);
```

Comment:
CONFIG_FULL_PATH

what was its value?

What is a configPath example value supposed to be?

I don't understand the change is my point @gentlementlegen

Reply:
We used to have only one configuration, `".github/.ubiquity-os.config.yml"` but now we have two (could eventually have more later on). The path is used to construct the url pointing to the configuration file that you can see in the errors. Because now we have multiple files, it was pointing to `".github/.ubiquity-os.config.yml"` always even if that was `".github/.ubiquity-os-dev.config.yml"` that actually had errors which was misleading.